controllers/past-grants/views/index.njk

```diff
@@ -78,7 +69,7 @@
                     </p>
                 </div>
 
-                <div class="grants-search{% if meta.totalResults === 0 %} has-no-results{% endif %}" v-if="totalResults > 0">
```

Comment:
This class meant that if you reloaded a page with 0 results (with JS on) and cleared the filters, you'd still see no results as this class hid them.